docs/understanding-radicle/how-radicle-works.md

```diff
@@ -0,0 +1,316 @@
+project A and project B are meant to converge someday. Are their histories and `HEAD` meant to be the same? Can you tell which repository is meant to be canonical?
+
+All this uncertainty happens because source code is *portable*—it can be cloned, edited, and hosted somewhere else
+without the project's maintainer knowing about it.
+
+Centralized code collaboration platforms like GitHub and GitLab create also create identities for the projects they
+host. If you create two identical repositories on these platforms, their identities are `github.com/bob/project` and
+`gitlab.com/bob/project`. You might know they're the same, but to someone else, there's no guarantee that the projects
+are the same despite seemingly sharing a `bob/project` identity.
+
+With centralized platforms, the code remains portable, but the *identity is non-portable*.
+
+A portable Radicle identity solves this problem because you always own your Radicle identity. Your identity might be replicated across the Radicle network, but because your key signs it, it's always yours. Your project may be replicated across the Radicle network, but because your identity is a delegate, only you can change the canonical remote/branch.
+
+Radicle identities are perfect for a peer-to-peer protocol because *the data is shipped with the source code, but not in
+the source code*.
+
+### Delegates
+
+Aside from is portable identity, the most important part of a project is the delegates.
+
+When you create a project, you add your identity as a delegate. Every time you push new changes to the project, they're
+signed with your Radicle identity and your key.
+
+When a project has multiple contributors, all of whom may be pushing code and patches to their individual remotes, the
+delegate's remote is considered the "canonical" version of the project's code base.
+
+## Radicle network
+
+The Radicle network is the web of connections between users (via their identities, devices, and projects) and protocols.
+No one owns this network, and it's all built on open-source code.
+
+As we continue developing the [peer-to-peer protocol](#peer-to-peer-protocol) that enables decentralization via Git
+gossip and replication, all your interactions with the Radicle network happen via seed nodes.
+
+### Seed nodes
+
+Seed nodes are the data availability layer for the Radicle network, designed to ensure that even if everyone in your
+peer-to-peer network is offline, you can always retrieve Radicle project data from a seed node. Seed nodes store your
+identity, but not your private keys, which means they can't take action on your behalf.
+
+Seed nodes are based on fully [open-source
+code](https://app.radicle.network/alt-clients.radicle.eth/rad:git:hnrkk9c4zt9thuxhwi1ukxqcrs5tmhbtcsony/tree/a19cdcb846a8360d8f55c10aac8175368881a6ee),
+and Radicle currently maintains three seed nodes &mdash; willow.radicle.garden, pine.radicle.garden, and
+maple.radicle.garden &mdash; with identical functionality.
```

Comment:
We could also link to the seeds: https://app.radicle.xyz/seeds/maple.radicle.garden